package.json

```diff
@@ -59,14 +60,19 @@
     "slug": "^0.9.1",
     "source-map-support": "^0.4.0",
     "vinyl-buffer": "^1.0.0",
-    "vinyl-source-stream": "^1.1.0"
+    "vinyl-source-stream": "^1.1.0",
+    "web-push": "^1.0.2"
```

Comment:
It'd be better if we could update to `web-push` version 2 and use the new interface:
```
webPush.sendNotification(endpointURL, {
  userPublicKey: key,
  userAuth: auth,
  payload: payload,
});